Sources/aura/utils/Decibel.hx

```diff
@@ -0,0 +1,25 @@
+package aura.utils;
+
+/**
+    The decibel (dB) is a logarithmic unit used to express the ratio of two values of a physical quantity.
+    One of these values is often a standard reference value, in which case the decibel is used to express the level[a] of the other value relative to this reference.
```

Comment:
What does the [a] in level[a] refer to?

Reply:
Good question.
One has to find the answer in the wikipedia article history :|